.circleci/config.yml

```diff
@@ -44,8 +44,7 @@ jobs:
           command: |
             ./cc-test-reporter format-coverage -t lcov ./coverage/lcov.info
             ./cc-test-reporter upload-coverage
-
 workflows:
   Test_and_Coverage:
     jobs:
-      - Test_and_Send_Coverage
+      - Test_and_Send_Coverage
```

Comment:
Add a new empty line at the end of every file

Reply:
done